packages/provider/src/wallet.ts

```diff
@@ -728,6 +732,20 @@ export const useWallet = () => {
   return walletInstance
```

Comment:
what's the point of this method existing?
it's the same as getWallet, but not safe, because it will sometimes return `undefined`

Reply:
Agree, `useWallet` just confuses it by having multiple use for similar/same thing